corelib-db/src/test/java/eu/europeana/corelib/solr/service/mock/SearchServiceMock.java

```diff
@@ -216,5 +216,12 @@ public boolean isHierarchy(String nodeId) {
         return false;
     }
 
-    
+    @Override
+    public Integer calculateTag(Integer mediaType, String mimeType, String imageSize, Boolean imageColor,
+                                Boolean imageGrayScale, String imageAspectRatio, String imageColorPalette, Boolean soundHQ,
+                                String soundDuration, Boolean videoHQ, String videoDuration) {
+          throw new NotImplementedException();
```

Comment:
Keep Mock objects as simple as possible. We still need to rewrite this mocks to use Mockito instead but for now, just return null instead of an NIY Exception.